<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>653</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>596</width>
    <height>653</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BW_CheckWINCC_1.7</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>BenewakeLogo.ico</normaloff>BenewakeLogo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-12</y>
      <width>629</width>
      <height>93</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_serial">
         <property name="text">
          <string>串口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_serial"/>
       </item>
       <item>
        <widget class="QLabel" name="label_baud">
         <property name="text">
          <string>波特率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_baud">
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>256000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>460800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>921600</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_port">
         <property name="text">
          <string>接口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_port">
         <item>
          <property name="text">
           <string>UART</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>IIC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RS485</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RS232</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_connect">
         <property name="text">
          <string>连接</string>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_refresh">
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>41</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget1" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>49</y>
      <width>591</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>591</width>
      <height>561</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButton_check">
     <property name="geometry">
      <rect>
       <x>164</x>
       <y>508</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>一键执行</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_return">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>0</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_return</zorder>
    <zorder>pushButton_check</zorder>
   </widget>
   <zorder>frame</zorder>
   <zorder>widget</zorder>
   <zorder>widget1</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>语言</string>
    </property>
    <addaction name="actionChinese"/>
    <addaction name="separator"/>
    <addaction name="actionEnglish"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionChinese">
   <property name="text">
    <string>中文</string>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>文档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>trigger_actOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_connect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>connectSerial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_refresh</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>refreshSerial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_check</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>checkAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>649</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChinese</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>trigger_actChinese()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEnglish</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>trigger_actEnglish()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHelp</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>trigger_actHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>trigger_actOpen()</slot>
  <slot>getSerialPort()</slot>
  <slot>connectSerial()</slot>
  <slot>refreshSerial()</slot>
  <slot>pollBaudrate()</slot>
  <slot>checkCommunication()</slot>
  <slot>checkAll()</slot>
  <slot>trigger_actChinese()</slot>
  <slot>trigger_actEnglish()</slot>
  <slot>trigger_actHelp()</slot>
 </slots>
</ui>
